scrooge-sbt-plugin/src/main/scala/com/twitter/ScroogeSBT.scala

```diff
@@ -210,13 +221,16 @@ object ScroogeSBT extends Plugin {
   )
 
   val packageThrift = mappings in (Compile, packageBin) ++= {
-    (if (scroogePublishThrift.value) (scroogeThriftSources in Compile).value else Nil).map { file =>
+    (if ((scroogePublishThrift in Compile).value) (scroogeThriftSources in Compile).value else Nil).map { file =>
```

Comment:
I'm not super familiar with the sbt dag stuff, does forcing the value of scroogePublishThrift make it harder for sbt to compile the dag correctly?  Isn't there a fishbone operator that we can use for that kind of thing?